aio/aio-proxy/aio_proxy/response/formatters/convention_collective.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+
+def extract_list_idcc_by_siren_from_ul(ul_result):
```

Comment:
Could be shorter : 
`extract_list_idcc_by_siren_from_ul` -> `extract_list_idcc_from_ul(`

Reply:
A bit complicated since within the ul object there is a `list_idcc` and a `list_idcc_by_siren` which are different fields.

Reply:
I am not sure to understan the `by_siren` within the ul. Shouldnt it be `by_siret` ?

Reply:
Yes 😅

Reply:
I have a branch to fix it in infra.

Reply:
Oh I see. It changes everything I just read